vertical-pod-autoscaler/README.md

```diff
@@ -139,15 +139,21 @@ There are four modes in which *VPAs* operate:
   them on existing pods using the preferred update mechanism. Currently, this is
   equivalent to `"Recreate"` (see below). Once restart free ("in-place") update
   of pod requests is available, it may be used as the preferred update mechanism by
-  the `"Auto"` mode. **NOTE:** This feature of VPA is experimental and may cause downtime
-  for your applications.
+  the `"Auto"` mode. **NOTE:** This feature of VPA may cause downtime for your
+  applications if new pods can't start (for example because there is no space for them
+  in the cluster). Using
+  [Cluster Autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler#cluster-autoscaler)
+  should mitigate the problem.
```

Comment:
I wouldn't say "mitigate" here - CA might help by creating a new Node but there are also scenarios when CA won't help:
* no more available capacity (e.g. NodePool at max size),
* selectors not allowing to use big enough Nodes (VPA only looks at the largest Node in the cluster),
* running out of resource quota,
* any webhooks affecting recreated Pods.

I don't think we want to go into so much detail here but maybe it's worth to mention the above in Known limitations or Caveats?

Reply:
Done